direct-query-core/src/main/java/org/opensearch/sql/prometheus/client/PrometheusClient.java

```diff
@@ -99,4 +99,12 @@ Map<String, List<MetricMetadata>> getAllMetrics(Map<String, String> queryParams)
    * @throws IOException If there is an issue with the request
    */
   JSONArray getAlertmanagerSilences() throws IOException;
+
+  /**
+   * Creates a silence in Alertmanager.
+   *
+   * @return JSONArray containing the silences
+   * @throws IOException If there is an issue with the request
+   */
+  String createAlertmanagerSilences(String silenceJson) throws IOException;
```

Comment:
return annotation doesn't match return type, is it JSONArray as string?

Reply:
Good catch, will update the annotation to be of String.